core/domain/skill_domain.py

```diff
@@ -357,10 +386,21 @@ def validate(self):
                     explanation)
 
 
+class WorkedExampleDict(TypedDict):
+    """Dictionary representing the WorkedExample domain object"""
+
+    question: Dict[str, state_domain.SubtitledHtml]
+    explanation: Dict[str, state_domain.SubtitledHtml]
```

Comment:
Should this be placed into _state_domain.py_

Reply:
ditto for some other dicts, if the domain object is in _state_domain.py_ then the dict should be there too.